packages/fxa-shared/test/db/models/auth/metadata.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS `dbMetadata` (
```

Comment:
I noticed this last time when I added some changes to the shared db, we would have to make sure that the table changes get reflected in two spots, the migration and here. I can't think of a simple way to avoid the duplication.

Reply:
Yes, this tripped me up a bit. This was the quick way of working around the problem. 

Part of me wonders if we should auto generate most of the `/test/**/*.sql` files with a script based on the state of the current db. In the past I've done something like this by using `mysqldump`. I like the auto generation approach, because it's more DRY and gives a clear view into the current database state.